src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -0,0 +1,140 @@
+                if (Objects.equals(key, NAME_IDENTIFIER)) {
+                    tempStr += String.format("%s. %s.\n", index, personRelevantInfo.get(NAME_IDENTIFIER));
+                } else if (Objects.equals(key, PHONE_IDENTIFIER)) {
+                    tempStr += generateNiceStr(personRelevantInfo, index, PHONE_IDENTIFIER);
+                } else if (Objects.equals(key, EMAIL_IDENTIFIER)) {
+                    tempStr += generateNiceStr(personRelevantInfo, index, EMAIL_IDENTIFIER);
+                } else if (Objects.equals(key, ADDRESS_IDENTIFIER)) {
+                    tempStr += generateNiceStr(personRelevantInfo, index, ADDRESS_IDENTIFIER);
+                } else if (Objects.equals(key, SALARY_IDENTIFIER)) {
+                    tempStr += generateNiceStr(personRelevantInfo, index, SALARY_IDENTIFIER);
+                } else if (Objects.equals(key, CLAIM_BUDGET)) {
+                    tempStr += generateNiceStr(personRelevantInfo, index, CLAIM_BUDGET);
+                } else if (Objects.equals(key, DEPARTMENT)) {
+                    tempStr += generateNiceStr(personRelevantInfo, index, DEPARTMENT);
+                } else {
+                    tempStr += generateNiceStr(personRelevantInfo, index, BIRTHDAY);
+                }
+            }
+            finalString += tempStr + "\n";
+        }
+        return new CommandResult(finalString);
+    }
+
+    /**
+     * Returns HashMap which contains employee's information.
+     *
+     * @param person Person object that we want to retrieve information from.
+     * @return HashMap, which informs us of the person's primary attributes.
+     */
+    private HashMap<String, String> generateArrInfo(Person person) {
+        HashMap<String, String> map = new HashMap<>();
+        map.put(NAME_IDENTIFIER, person.getName().toString());
+        map.put(PHONE_IDENTIFIER, person.getPhone().toString());
+        map.put(EMAIL_IDENTIFIER, person.getEmail().toString());
+        map.put(ADDRESS_IDENTIFIER, person.getAddress().toString());
+        map.put(SALARY_IDENTIFIER, person.getSalary().toString());
+        map.put(CLAIM_BUDGET, person.getClaimBudget().toString());
+        map.put(DEPARTMENT, person.getDepartment().toString());
+        map.put(BIRTHDAY, person.getDob().toString());
+        return map;
+    }
+
+    /**
+     * Returns String containing relevant person's information.
+     *
+     * @param personInfo HashMap containing Person's primary attributes.
+     * @param index Integer which represents index from list.
+     * @param identifier String which represents what attribute to view from.
+     * @return String representation of relevant information.
+     */
+    public String generateNiceStr(HashMap<String, String> personInfo, int index, String identifier) {
+        String result = personInfo.get(identifier);
+        String name = personInfo.get(NAME_IDENTIFIER);
+        return String.format("%s. %s's %s is %s.\n", index, name, identifier, result);
```

Comment:
when the command is "view b/1,2 s/1" will the list numbering be 1. 2. 1.? Not sure what index and indexlist refer to in this code